### ISSUE TITLE
elementExplorer cuts of the first letter of my statement

### ISSUE THREAD

**Issue by pavlinap** [angular/protractor#1970]
When I start elementExplorer regardless of the input I get an error telling me that there is a reference error
```
> element(by.id(''));
ReferenceError: evalmachine.<anonymous>:1
lement(by.id(''))
lement is not defined
```

![issue](https://cloud.githubusercontent.com/assets/1632422/6823932/4129fbf2-d2ed-11e4-86fd-b6cbe6743e0f.png)

**Comment by juliemr** [comment]
Dupe of https://github.com/angular/protractor/pull/1898 - @hankduan can we fix the text parsing issue separately from the timing issue?

**Comment by hankduan** [comment]
Yes, feel free to review the PR. It will fix the node 0.12.0 compatibility issue. (preferable if you can review https://github.com/angular/protractor/pull/1954 before https://github.com/angular/protractor/pull/1898 though so tests are in place)

**Comment by bootstraponline** [comment]
:+1: elementExplorer is broken on node v0.12.2.

**Comment by sunitavc** [comment]
Will this work if I downgrade node? I'm facing the same issue and I have node 0.12.0

**Comment by bootstraponline** [comment]
It works for me on node v0.10.30.

**Comment by sibeliusseraphini** [comment]
+1

**Comment by hankduan** [comment]
@juliemr can you review that PR when you get a chance next week?

**Comment by ralberts** [comment]
+1 Is the workaround to downgrade node versions?

**Comment by bootstraponline** [comment]
> Is the workaround to downgrade node versions?

Yes, to 0.10.30.

**Comment by juliemr** [comment]
This is fixed now in 2.1.
